distributed/scheduler.py

```diff
@@ -196,21 +195,26 @@ def _to_dict_no_nest(self, *, exclude: "Container[str]" = ()) -> dict:
 
 class MemoryState:
     """Memory readings on a worker or on the whole cluster.
+    See :doc:`worker-memory`.
```

Comment:
This class is now documented in Sphinx.

Reply:
Historically we've kept a newline between the title line of a docstring and the body.  Thoughts on this?  Any objection to continuing with it?

I don't care so much about this particular instance (please don't block on a comment this small), but more broadly I'd love to align on this if it's easy.

https://docs.dask.org/en/stable/develop.html#docstrings